docs/source/requirements.rst

```diff
@@ -34,6 +34,10 @@ Kokkos 4.x
       * 10.0.0
       * 12.0.0, 14.0.0
 
+    * * Clacc (OpenACC) (experimental)
+      * 40cad5ea5c42
+      * 40cad5ea5c42
```

Comment:
is this a copy/paste error?

Reply:
No. this is a commit number; Clacc (https://github.com/llvm-doe-org/llvm-project/tree/clacc/main) is a customized version of the upstream LLVM, and thus I used the Clacc's commit number instead of the Clang version number not to be confused with the upstream Clang/LLVM compiler.

Reply:
I would just drop `Clacc` here. We don't test with it in our `jenkins` CI anyway and if there is no release it's a limited use for the users targeted here.

Reply:
I meant that it lists the same commit hash twice

Reply:
@masterleinad Should it be added to the test matrix, then?

Reply:
> @masterleinad Should it be added to the test matrix, then?

That's something to discuss.

Reply:
The entry for Clacc is removed.